access-control-app/int-test/cmr/access_control/int_test/access_control_group_crud_test.clj

```diff
@@ -103,8 +103,10 @@
 
     (testing "Create group with fields at maximum length"
       (let [group (into {} (for [[field max-length] field-maxes]
-                             [field (string-of-length max-length)]))]
-        (is (= 200 (:status (u/create-group (e/login (u/conn-context) "user1") group)))))))
+                             [field (string-of-length max-length)]))
+            ;; FIXME understand why having this outside of a binding causes JVM corruption error
+            ;; in Clojure 1.10.0
+            _ (is (= 200 (:status (u/create-group (e/login (u/conn-context) "user1") group))))])))
```

Comment:
We probably want to keep track of the issues around `is`that have been discovered in 1.10:
 * https://dev.clojure.org/jira/browse/CLJ-2458

Reply:
Note that without the `is` it also throws the JVM corruption issue on the call to create-group.